package.json

```diff
@@ -4,7 +4,8 @@
   "description": "_17th September 2022_",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npx nodemon server.js"
```

Comment:
@karanja-joseph nice thought instead of using `nodemon` every time.

Reply:
Thanks!